src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,167 @@
+package duke.ui;
+
+import duke.model.Goal;
+import duke.model.Meal;
+import duke.model.user.User;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    private static final String padding = "     ";
+    private static final String boundary = "    ____________________________________________________________";
```

Comment:
constants naming violation. please check the NUS OSS coding standards and follow them strictly